apps/web/src/app/[lang]/app/[type]/crm/[data]/(alternative)/[id]/form.tsx

```diff
@@ -156,10 +148,9 @@ export default function Form({
         toast.error(response.status);
         return;
       }
-      const _data =
-        response.data as Volo_Abp_Application_Dtos_PagedResultDto_18;
+      const _data = response.data as PagedResultType;
```

Comment:
we can remove the PageResultType after we handle the `response.type === "api-error"`

```suggestion
      const _data = response.data;
```